inlineplz/decorators.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import
+from __future__ import print_function
+
+from functools import partial
+import inspect
+from inlineplz.util.system import run_command, install_linter
+from inlineplz.registry import register_linter
+
+
+class Registry(object):
+    REGISTRY = dict()
+    LINTERS = []
+
+    @staticmethod
+    def register(klass):
+        kprops = {n: m for (n, m) in inspect.getmembers(klass)}
+        config = {
+            "name": klass.NAME,
+            "install": kprops.get("INSTALL", None),
```

Comment:
isn't `None` the default return value for `.get`?

Reply:
yep, I'll take it out